clang/test/Driver/riscv-cpus.c

```diff
@@ -403,6 +403,11 @@
 // MCPU-MARCH: "-nostdsysteminc" "-target-cpu" "sifive-e31" "-target-feature" "+m" "-target-feature" "+c"
 // MCPU-MARCH: "-target-abi" "ilp32"
 
+// march=unset erases previous march
```

Comment:
`/// -march=unset erases previous -march`.

Always include `-` when referring to a `-` option.